doc/source/reference/apis/metadata.md

```diff
@@ -15,18 +15,23 @@ Incubating update 1:
 - we added `v1` format that better describes current SeldonMessage
 - definition through environmental variables now accepts both `yaml` and `json` input
 
+Incubating update 2:
```

Comment:
I think we can now remove all incubating updates and incubating references as this (master) is 1.3 😀